### DISCUSSION TITLE
iOS iNav Configurator

### DISCUSSION THREAD

**Discussion by Ultraexpert** [iNavFlight/inav#8313] [Q&A]
Would love to port the iNav configurator to iOS for use with Bluetooth and WiFi, like the Speedybee app. Just trying to gauge how hard this would actually be.

**Comment by timmo** [comment]
would love to use iNav on my iOS devices!

**Comment by sensei-hacker** [comment]
It would be cool if you want to try doing that.
It's my understanding that that IOS doesn't support Electron apps, so there would be some work to do.

90%+ of the code is html and JavaScript which could run  mostly unchanged on top of some mobile framework such as Capacitor. The connections to the FC (USB, tcp, Bluetooth) would probably need to be reworked.

It might be almost the same work to convert it to a web app that runs on _any_ device. Maybe that's something to consider.

**Comment by beymaxios** [comment]
https://github.com/iNavFlight/inav/discussions/11407#discussion-9593126

Please watch this
